test/parallel/test-child-process-bad-stdio.js

```diff
@@ -2,9 +2,12 @@
 // Flags: --expose-internals
 const common = require('../common');
 
-if (process.argv[2] === 'child') {
+if (process.argv[2] === 'timeout') {
   setTimeout(() => {}, common.platformTimeout(100));
   return;
+} else if (process.argv[2] === 'interval') {
+  setInterval(() => {}, 10_000);
+  return;
 }
```

Comment:
Just a nit, feel free to ignore.

```suggestion
}

if (process.argv[2] === 'interval') {
  setInterval(() => {}, 10_000);
  return;
}
```